LoggingKata/TacoBell.cs

```diff
@@ -0,0 +1,78 @@
+﻿using GeoCoordinatePortable;
+using System;
+using System.Collections.Generic;
+
+namespace LoggingKata
+{/// <summary>
+/// TacoBell object with Name, Longitude and Latitude
+/// </summary>
+///
+    public class TacoBell : ITrackable
+    {
+        public string Name { get; set; }
+        public Point Location { get; set; }
+        public TacoLogger Log { get; set; }
+
+        public static List<TacoBell> GetTacoBells = new List<TacoBell>();
+
+        public TacoBell(string name, Point loc)
+        {
+            this.Name = name;
+            this.Location = loc;
+            this.Log = new TacoLogger();
+        }
+
+        public TacoBell(string[] line)
+        {
+            TacoBell taco = TacoParser.ParseLine(line);
```

Comment:
There's something unusual about this.  We're within a constructor for the `TacoBell` class, yet we're getting a new instance of a `TacoBell` from another method and using it.  Could this be re-designed?